bc/subscription/utils/courtlistener.py

```diff
@@ -67,6 +68,20 @@ def lookup_docket_by_cl_id(cl_id: int):
     return response.json()
 
 
+def lookup_document_by_doc_id(cl_doc_id: int | None):
```

Comment:
When would this be `| None`? What type does this function return?

Reply:
The `doc_id` won't be `None` for new records but I set the default value to `null` for old records (We need a default value because this table has data). 

Mypy was complaining about the type hint because the `doc_id` field is nullable in the `WebhookEvent` model and that's why I included `None` in the type hint of the helper function. 

This helper function returns a `dict` and I realized We can use the `fields` parameter to include just the `filepath_local` attribute in the response. The new request would be:

```python
response = requests.get(
        f"{CL_API['recap-documents']}{doc_id}/",
        params={"fields": "filepath_local"},
        headers=auth_header(),
        timeout=5,
    )
```

Using the previous request, the return type would be: `dict[str, str | None]`

Do you think We can use another approach to set a default value for previous records?